internal/server/http/handler/v1/treecluster/handler.go

```diff
@@ -43,9 +61,22 @@ func GetAllTreeClusters(svc service.TreeClusterService) fiber.Handler {
 			data[i] = treeClusterMapper.FromInListResponse(domain)
 		}
 
+		var pagination *entities.Pagination
+		if limit != -1 {
+			totalPages, nextPage, prevPage := entities.CalculatePagination(int32(totalCount), limit, page)
+
+			pagination = &entities.Pagination{
+				Total:       totalCount,
+				CurrentPage: page,
+				TotalPages:  totalPages,
+				NextPage:    nextPage,
+				PrevPage:    prevPage,
+			}
+		}
+
```

Comment:
With the fiber middleware, this will be put into a context and can be used everywhere in our app, that has this context